index.html

```diff
@@ -0,0 +1,129 @@
+      <h1 class="title">
+        Mawazine Music <br>
+      <span class="title-content">Festival</span> <br>
+      <span class="title-content">2022</span></h1>
+    </div>
+    <div class="content">
+      <p>The Mawazine Festival in Rabat, Morocco is a music festival of incredible scale. It has been named the largest festival in the world with the largest ratio of visitors per stage. The name “Mawazine” roughly translates to Rhythms of the World.</p>
+    </div>
+    <h3 class="event-date">2022.07.25(MON) ~ 27(WED)</h3>
+    <h4 class="event-location">@Agdal Rue 5 | Hilaton | Rabat</h4>
+  </main>
+  <section id="Program" class="main-section">
+   <div class="section-title">
+    <h2 >Main Program</h2>
+    <span class="bottom-title"></span>
+   </div>
+    <div class="cards">
+      <div class="card">
+        <img src="Assets/program_icon_01.png" alt="">
+        <h3>MMF 2022</h3>
+        <p>The Mawazine Festival first started in 2001 as a much smaller event dedicated to World Music. </p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_02.png" alt="">
+        <h3>MUSIC</h3>
+        <p>Appreciate various music creations, from different parts of Africa. Feel the culture from Millenium Hall..</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_03.png" alt="">
+        <h3>TICKETS</h3>
+        <p>Come a get your ticket from one of our agences and do not miss the amizing Show.</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_04.png" alt="">
+        <h3>WORKSHOPS</h3>
+        <p>Along with this big event that will be held in Rabat, there are some wonderful WORKSHOPS that we do not want to miss</p>
+      </div>
+      <div class="card">
+        <img src="Assets/program_icon_05.png" alt="">
+        <h3>MMF Ingite</h3>
+        <p>Get opportunities to network with the attendees and hear some professional advice from top artists.</p>
+      </div>
+    </div>
+    <button class="join" type="submit">Join MMF NOW</button>
+  </section>
+  <section class="speakers-sec" id="Geusts">
+    <h2>Special Geust</h2>
+    <span class="bottom-title"></span>
+    <article class="wrapper"></article>
+  </section>
+  <section class="partners-sec" id="Parteners">
+    <div class="section-title">
+      <h2>Panters</h2>
+      <span class="bottom-title"></span>
+     </div>
+      <div class="partners">
+        <img src="Assets/mtc-logo.png" alt="partner1" >
+        <img src="Assets/anghami.png" alt="partner2" >
+        <img src="Assets/pepwear-logo.png" alt="partner3" >
+        <img src="Assets/coke-studio-logo.png" alt="partner4" >
+        <img src="Assets/yamaha-logo.png" alt="partner5" >
+      </div>
+  </section>
+  <footer>
+    <div class="bottom">
+      <img class="foot-img" src="Assets/mawazine.png" alt="logo">
+      <p>
+        2022 Mawazine Music Festival <br>
+        <span>© Some rights reserved. </span>
+      </p>
+    </div>
+  </footer>
```

Comment:
Consider adjusting the height of your partner and footer section to match it with the original design.
| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182562134-916049c1-9f97-4f73-8a51-250408891381.png) | ![image](https://user-images.githubusercontent.com/33268927/182562295-9f8821c6-b7e6-4347-b967-ee8d1b0a6c49.png) |

Reply:
Also, this section is not included in the mobile version. Consider hiding it in the mobile version of the application.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/182566274-10b4ca42-c4c4-47ca-b52f-1e623f84ca2b.png) | ![image](https://user-images.githubusercontent.com/33268927/182566395-e9986fce-55c5-4789-b3fd-f76ec648e8e9.png) |